detox/android/detox/src/full/java/com/wix/detox/DetoxMain.kt

```diff
@@ -32,6 +32,7 @@ object DetoxMain {
         initCrashHandler(externalAdapter)
         initANRListener(externalAdapter)
         initReactNativeIfNeeded(rnHostHolder)
+        initBlacklistIfNeeded()
```

Comment:
Need to double check whether in fact the URL's must be set inherently in `initReactNative`, seeing that, if I'm not mistaken, RN init also includes an idle-wait.